flask_iiif/config.py

```diff
@@ -107,32 +107,35 @@
     "v1": {
         "region": {
             "ignore": "full",
-            "validate": "(^full|(pct:)?([\d.]+,){3}([\d.]+))",
+            "validate": r"(^full|(pct:)?([\d.]+,){3}([\d.]+))",
         },
         "size": {
             "ignore": "full",
             "validate": (
-                "(^full|[\d.]+,|,[\d.]+|pct:[\d.]+|[\d.]+," "[\d.]+|![\d.]+,[\d.]+)"
+                r"(^full|[\d.]+,|,[\d.]+|pct:[\d.]+|[\d.]+,[\d.]+|![\d.]+,[\d.]+)"
```

Comment:
:factory_worker:

Reply:
AFAIK those are artifacts from the migration to black formatting :black_cat: